dropwizard-jersey/src/main/java/io/dropwizard/jersey/validation/JerseyViolationExceptionMapper.java

```diff
@@ -0,0 +1,29 @@
+package io.dropwizard.jersey.validation;
+
+import com.google.common.base.Function;
+import com.google.common.collect.FluentIterable;
+import com.google.common.collect.ImmutableList;
+
+import javax.validation.ConstraintViolation;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.ext.ExceptionMapper;
+import java.util.Set;
+
+public class JerseyViolationExceptionMapper implements ExceptionMapper<JerseyViolationException> {
```

Comment:
I like the fact that now it'll be possible to intercept validation exceptions via an exception mapper and write a custom response for them.  This is something that I've struggled with in the past where I've wanted custom formatting of validation errors